index.html

```diff
@@ -20,11 +20,52 @@
 
```

Comment:
- [ ] The project should contain a `Home` page and `About` page, kindly make sure to include your about page in the project so it's complet